backends/google/drive/google-drive.js

```diff
@@ -228,7 +228,8 @@ export default class GoogleDrive extends Google {
 	}
 
 	static urls = [
-		{hostname: "drive.google.com", pathname: "/file/d/:id"},
+		{hostname: "drive.google.com", pathname: "/file/d/:id/*"},
+		{hostname: "drive.google.com", pathname: "/drive/*/my-drive"},
 		{hostname: "drive.google.com", pathname: "/drive/folders/:folderId"},
 		{hostname: "drive.google.com", pathname: "/drive/*/folders/:folderId"},
```

Comment:
Pretty sure the last two can be combined somehow, but I didn't know enough about the URL structure of GDrive

Reply:
Will give this a closer look.

Reply:
Done!